docusaurus/docs/get-started/get-started.mdx

```diff
@@ -18,86 +18,118 @@ import InstallNPM from '@snippets/createplugin-install.npm.md';
 import InstallPNPM from '@snippets/createplugin-install.pnpm.md';
 import InstallYarn from '@snippets/createplugin-install.yarn.md';
 
-Grafana's plugin tools offer an officially supported way to extend Grafana's core functionality. We have designed these tools to help you to develop your plugins faster with a modern build setup and zero configuration.
-
-The plugin tools consist of two packages:
+# Get started
 
-- `create-plugin`: A CLI to scaffold new plugins or migrate plugins created with `@grafana/toolkit`.
-- `sign-plugin`: A CLI to sign plugins for distribution.
+Welcome to the world of Grafana plugin creation, where you can enhance Grafana's foundational features. In this guide, you'll learn how to get started by scaffolding a plugin, running it in an efficient development environment, and using its basic features.
 
 ## Quick Start
 
-To scaffold your plugin, follow these steps:
+Scaffold a new plugin with a single command! Run the following and answer the prompts:
 
-1. Run the following command:
+<CodeSnippets
```

Comment:
<img width="337" alt="Screenshot 2023-10-26 at 19 18 42" src="https://github.com/grafana/plugin-tools/assets/19860021/77bc0393-43c9-4dc6-a056-2464f4dc44cc">
<img width="400" alt="Screenshot 2023-10-26 at 19 18 40" src="https://github.com/grafana/plugin-tools/assets/19860021/cdac0e34-4320-4ef0-9d05-0070345f88fc">

Not sure if we can get npx highlighted in the same manner? @jackw

Reply:
They're coded the same way, so that is odd. @jackw ?